environment/toolchain/folly.diff

```diff
@@ -0,0 +1,2 @@
+31d30
```

Comment:
Maybe consider switching to patch files as I did for the updated v4 toolchain.

Reply:
For which projects? Do you mean git patch files?
As we get the archives, we don't get the git folder so `git apply` doesn't work.

Reply:
I meant something like [this](https://github.com/memgraph/memgraph/blob/T0804-MG-base-thrift-project/environment/toolchain/folly.patch), but this is not that important. I just copied the original source folder, made the changes and used `diff -ur`. If you name the original/modified folder as `a` and `b` then you get nice paths.

My reasoning to switch to patch files (not git patch, simple patch) is it can contain changes to multiple files, and also contain the path for the files to be modified.

Reply:
oh okay, I would leave it as is for this patch as it's only one file but I'm definitely for it when there are multiple of them.